src/wirecraft/server/server.py

```diff
@@ -69,3 +74,48 @@ def get_money(self) -> int:
 
     def buy_item(self, type: str) -> None:
         return
+
+    def add_cable(self, id_device_1: int, port_1: int, id_device_2: int, port_2: int, level: int) -> None:
+        statement = Cable(
+            id_device_1=id_device_1, port_1=port_1, id_device_2=id_device_2, port_2=port_2, id_level=level
+        )
+        with Session(engine) as session:
+            session.add(statement)
+            session.commit()
+            print("Cable added")
+
+    def end_cable(self, cable_id: int, device_id: int, port_id: int) -> None:
+        statement = select(Cable).where(Cable.id == cable_id)
+        with Session(engine) as session:
+            cable = session.exec(statement).one()
+            cable.id_device_2 = device_id
+            cable.port_2 = port_id
+            session.add(cable)
+            session.commit()
+
+    def get_level_cables(self, id_level: int) -> list[Cable]:
+        statement = select(Cable).where(Cable.id_level == id_level)
+        with Session(engine) as session:
+            return list(session.exec(statement).all())
```

Comment:
Is list needed ? .all() already return a Sequence, so I see no reason to use list[]